src/3rd_party-static/libusbx-1.0.16/CMakeLists.txt

```diff
@@ -10,21 +10,37 @@ include_directories (
   ./
 )
 
-set (LIBUSB_SOURCES
-	./libusb/core.c
-	./libusb/descriptor.c
-	./libusb/hotplug.c
-	./libusb/io.c
-	./libusb/strerror.c
-	./libusb/sync.c
-	./libusb/os/linux_udev.c
-	./libusb/os/linux_netlink.c
-	./libusb/os/linux_usbfs.c
-	./libusb/os/poll_posix.c
-	./libusb/os/threads_posix.c
-)
+if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+  include_directories (
+    ./msvc
+  )
+  set (LIBUSB_SOURCES
+    ./libusb/core.c
+    ./libusb/descriptor.c
+    ./libusb/hotplug.c
+    ./libusb/io.c
+    ./libusb/strerror.c
+    ./libusb/sync.c
+    ./libusb/os/poll_windows.c
+    ./libusb/os/threads_windows.c
+    ./libusb/os/windows_usb.c
+  )
+else()
```

Comment:
May be extract the common source and then add to LIBUSB_SOURCE platform-specific?